openstef_dbc/utils.py

```diff
@@ -57,3 +57,34 @@ def parse_influx_result(
     result.index.name = "datetime"
     result.columns.name = ""
     return result
+
+
+def round_time_difference(time_diff, options):
+    """Rounds a time difference to the first bigger option.
+
+    Used to calculate the tAhead for forecasts"""
+    next_bigger_option = next((opt for opt in options if opt > time_diff), None)
+    if next_bigger_option is None:
+        return time_diff
+    return next_bigger_option
+
+
+max_timediff = 1000
+
+
+def round_single_time_diff(time_diff, options):
+    """Rounds a time difference to the first bigger option.
+    Used to calculate the tAhead for forecasts"""
```

Comment:
Why would we want to round to the first bigger option? Would we not rather round to the first smaller option instead?

For example, if the time difference between the timestamp of the forecast and now is 15 hours, than I would rather have the assigned tAhead be 12 hours than 20 hours. 20 hours would be misleading since you only actually forecasted 15 hours ahead. On the other hand the 15 hour ahead forecast that would be labelled 12 hours ahead, will eventually be overwritten by a forecast that is actually made 12 hours ahead, since we make new forecasts every 15 minutes.